example/async-stream-client.rs

```diff
@@ -120,12 +120,17 @@ async fn sum_stream(cli: streaming_ttrpc::StreamingClient) {
 
 #[cfg(unix)]
 async fn divide_stream(cli: streaming_ttrpc::StreamingClient) {
+    let mut stream = cli
```

Comment:
nit: Could we create a test case to test the default value specifically instead of messing it into `divide_stream`?

Reply:
A new case was added instead of modify one of the exists.